grammars/stata-webdoc.cson

```diff
@@ -0,0 +1,32 @@
+'scopeName': 'source.webdoc.stata'
+'name': 'Stata Webdoc'
+'fileTypes': [
+  'do'
+  'dowd'
+]
+'firstLineMatch': 'webdoc init'
+'patterns': [
+  {
+    'begin': '(/\\*\\*\\*)(\\s)'
+    'beginCaptures':
+      '1':
+        'name': 'punctuation.definition.comment.webdoc.stata'
+    'end': '(^|\\s)(\\*\\*\\*/)'
+    'endCaptures':
+      '2':
+        'name': 'punctuation.definition.comment.webdoc.stata'
+    'name': 'meta.tag.block.$2.html'
+    'patterns': [
+      {
+        'include': 'text.html.basic'
```

Comment:
Shouldn't this include Markdown and not HTML?

Reply:
By default webdoc takes and outputs html. Markdown would have to be specified (`webdoc init, md`).

Reply:
It's not possible to know whether the user added the `md` option or not to the `webdoc init` command. But since HTML is valid Markdown, there isn't really much of a point to include `text.html` and not the Markdown grammar here. Whether you write in Markdown or with HTML tags it'll get highlighted when you include the Markdown grammar.

Reply:
I agree, changed it to `text.md`